client/state/reader/tags/images/selectors.js

```diff
@@ -16,7 +11,7 @@ import 'state/reader/init';
  * @returns {object} Image
  */
 export function getTagImages( state, tag ) {
-	const items = get( state, 'reader.tags.images.items' );
+	const items = state?.reader?.tags?.images?.items;
```

Comment:
Assuming that `reader` reducer has been loaded, `tags`, its `images`, and respectively its `items` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?